srcpkgs/mb2md/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'mb2md'
+pkgname=mb2md
+version=3.20
+revision=1
+build_style=fetch
+depends="perl-TimeDate"
+short_desc="Convert mbox mailboxes in maildir mailboxes"
```

Comment:
i believe it is `into maildir mailboxes`

Reply:
yeah, my bad.
I pushed new patch to fix it.